### ISSUE TITLE
[P1][Sourcing][Accessibility][chooser]Multi-input input field does not convey selected values / No hint for Screen reader users when focusing the input field with TAB key about the values in the field

### ISSUE THREAD

**Issue by I543348** [SAP/fundamental-ngx#10260]
#### Is this a bug, enhancement, or feature request?
Bug

#### Briefly describe your proposal.
Blind users wont understand when they immediately reach multi-input element's search input field using keyboard. This is the first area which receives focus , but the screenreader does not announce whether there are some already selected values or not.
This will be helpful for disabled users to understand if the user has to select freshly from list or some are selected already.

#### Which versions of Angular and Fundamental Library for Angular are affected? (If this is a feature request, use current version.)
Angular 15
0.43.0-rc.16

#### If this is a bug, please provide steps for reproducing it.
1) Bring the focus on the multi0input using tab key for the first time
2) check the attached screenreader output.
3) Observe that there is nothing read to users how many tokens are selected or if nothing is selected
<img width="1792" alt="Screenshot 2023-08-01 at 3 21 50 PM" src="https://github.com/SAP/fundamental-ngx/assets/88182358/e561361c-ca44-4f52-b6b0-409032827b1b">

**Comment by I543348** [comment]
ATL comment:

No hint for Screen reader users when focusing the input field with TAB key about the values in the field.

**Comment by nikolay-kolarov** [comment]
Hi,

I tested with two other screen readers (Jaws and NVDA) and I got the announcement "Contains 3 token".
I also checked that the text is correctly referenced in the aria-describedby attribute in the input element.

So VoiceOver is not reading the description for some reason and it might be a specific VoiceOver behavior.
It can be researched and checked if there is a specific shortcut or some configuration that will force the announcement on the description. But on code level, the implementation seems to be ok and the info is added.

Best Regards,
Nikolay

**Comment by droshev** [comment]
@I543348 which screen reader did you use in your testing?

**Comment by sagar7259** [comment]
Hello,

The issue is still reproducible in JAWS and NVDA. Here on accessing the multi input its corresponding information regarding that how many tokens present within this should also be associated when read. 
Here is the example of multi input of UI5 where the information "contains x tokens" is being read correctly.

https://sapui5.hana.ondemand.com/#/entity/sap.m.MultiInput/sample/sap.m.sample.MultiInputMaxTokens

Regards,
Sagar

**Comment by nikolay-kolarov** [comment]
Hi,

Thanks for the info. The mobile example only showcases how this behaves on mobile, so it is not relevant for screen reader testing on desktop.
As it was tested on the regular examples all works fine, so I am closing this one.

@droshev  - FYI

Best Regards,
Nikolay

**Comment by sagar7259** [comment]
@nikolay-kolarov this issue is not only related to mobile example but also holds true for the desktop examples.

https://sap.github.io/fundamental-ngx/#/core/multi-input

All other examples of desktop multi value inputs has this issue, hence this needs to be fixed.

@rvamudha please reopen this ticket.

Regards,
Sagar

**Comment by nikolay-kolarov** [comment]
Hi,

When I tested it there was such an announcement (e.g. "Contains 3 tokens"). But not is seems to be missing.

@droshev  - Could you please have a look?

Thanks,
Nikolay

**Comment by droshev** [comment]
> @nikolay-kolarov this issue is not only related to mobile example but also holds true for the desktop examples.
> 
> https://sap.github.io/fundamental-ngx/#/core/multi-input
> 
> All other examples of desktop multi value inputs has this issue, hence this needs to be fixed.
> 
> @rvamudha please reopen this ticket.
> 
> Regards, Sagar

Can you test the issue on version `0.43.9`?

**Comment by droshev** [comment]
@N1XUS there is a small differences between core multi-input and platform multi-input. the latter has info about the number of the selected tokens

Platform: 
<img width="864" alt="Screenshot 2023-09-14 at 4 11 08 PM" src="https://github.com/SAP/fundamental-ngx/assets/4380815/8f70c80f-03b5-4b59-b9a8-2c415ef71c8a">


Core:
<img width="1019" alt="Screenshot 2023-09-14 at 4 11 20 PM" src="https://github.com/SAP/fundamental-ngx/assets/4380815/a54ada25-e0f4-4766-83c1-133e1c2f9e3a">


what we are missing in core:
```
<span aria-hidden="true" class="fdp-multi-input__invisible-text" ....>Contains X tokens</span>
```
next to the tokenizer. Can you bring that part in core multi input? Thanks

**Comment by sagar7259** [comment]
Hello,

The reported issue is still not fixed. On entering into the input only "Multi value input edit type in text" is being read. 

The expected reading is "Multi value input edit has 3 tokens type in text"
Kindly look into it.

@rvamudha please reopen this ticket.

Regards,
Sagar

**Comment by droshev** [comment]
@sagar7259 did you test with `0.43.10`?
